dev/connector.js

```diff
@@ -0,0 +1,221 @@
+                    let value = account[key];
+                    accounts[0][key] = value;
+                }
+            } else {
+                var accounts = [account];
+            }
+
+            if (konnector) {
+                return konnector.updateAttributes({accounts}, callback);
+            } else {
+                return callback(new Error(
+                    "Can't find given konnector (slug expected).")
+                );
+            }
+        });
+    },
+
+
+    // Display konnector information stored in the database. It helps debugging
+    // by ensuring that values are properly set.
+    display(konnectorName, callback, log) {
+        let Konnector = require(
+            path.join(
+                baseBuildPath,
+                '/server/models/konnector'
+            )
+        );
+        return Konnector.all(function(err, konnectors) {
+            if (konnectorName) {
+                var konnector = konnectors.find(konnector => konnector.slug === konnectorName);
+                if (konnector) {
+                    displayKonnector(konnector, log);
+                } else {
+                    log.error(`Can't find konnector ${konnectorName}.`);
+                }
+            } else {
+                konnectors.sort(function(a, b) {
+                    if (a.slug != null) {
+                        return a.slug.localeCompare(b.slug);
+                    } else if (a.date != null) {
+                        return a.date > b.date;
+                    } else {
+                        return a.id > b.id;
+                    }
+                });
+                for (var konnector of konnectors) { displayKonnector(konnector, log); }
+            }
+            return callback();
+        });
+    },
+
+
+    /**
+     * Initialize connectors
+     */
+    init(callback, log) {
+        let initKonnectors = require(
+            path.join(
+                baseBuildPath,
+                '/server/init/konnectors'
+            )
+        );
+        return cozydb.configure(
+            {
+                modelsPath: path.join(baseBuildPath, '/server/models')
+            },
+            null,
+            () => initKonnectors(() => callback())
+        );
+    },
+
+    /**
+     * Deinitialize connectors
+     */
+    deinit(konnectorName, callback, log) {
+        let Konnector = require(
+            path.join(
+                baseBuildPath,
+                '/server/models/konnector'
+            )
+        );
+        return Konnector.all(function(err, konnectors) {
+            if (err) { return log.error(err); }
+
+            if (konnectorName) {
+                let konnector = konnectors.find(konnector => konnector.slug === konnectorName);
+                return konnector.destroy(callback);
+            } else {
+                konnectors.map(konnector => konnector.destroy(callback));
+            }
+        });
+    }
+};
+
+
+function __guard__(value, transform) {
```

Comment:
Is this one really useful or can we write its call in a more smart way to remove it?